__tests__/utils.test.js

```diff
@@ -102,3 +104,26 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
```

Comment:
Well done for testing your Util here, however, I'm not sure this is completely necessary, as you could instead just require in the JSON in your test in order to compare it.